_episodes/03-configuration.md

```diff
@@ -3,91 +3,178 @@ title: "Configuration"
-Python toolkits such as cartopy will attempt to download data files at run
-time, typically geographic data files such as coastlines or land surface maps.
-This can fail if the machine does not have access to the wider internet. This
-location allows us to tell cartopy (and other similar tools) where to find the
-files if they can not be downloaded at runtime. To reiterate, this setting is
-not for model or observational datasets, rather it is for data files used in
-plotting such as coastline descriptions and so on.
+## Directory structure for the data from different projects
 
+Input data can be from various models, observations and reanalysis data that adhere to the [CF/CMOR standard](https://cmor.llnl.gov/). To set a directory, you can use one of the values of ``default``, ``BADC``, ``DKRZ``, ``ETHZ``, .... Let's use ``default`` in our example:
 
-## Output settings
+~~~YAML
+drs:
+  CMIP5: default
+~~~
+
+> ## Available drs
+>
+> For more information about directories, you can visit ESMValTool [documentation](https://esmvaltool.readthedocs.io/projects/esmvalcore/en/latest/esmvalcore/config.html#developer-configuration-file).
+{: .callout}
+
+## Number of parallel tasks
 
-.. code-block:: yaml
+This option enables you to perform parallel processing. You can choose the number of tasks in parallel as 1/2/3/4/... or you can set it to ``null`` that tells ESMValTool to use the number of available CPUs:
 
-  # Diagnostics create plots? [true]/false
-  write_plots: true
-  # Diagnositcs write NetCDF files? [true]/false
-  write_netcdf: true
+~~~YAML
 
-The ``write_plots`` setting is used to inform ESMValTool about your preference
-for saving figures. Similarly, the ``write_netcdf`` setting is a boolean which
-turns on or off the writing of netCDF files.
+max_parallel_tasks: null
+~~~
 
-The ```rootpath`` specifies the directories where ESMValTool will look for input
-data. Similarly, ``output_dir`` specifies where ESMValTool will store its
-output, i.e. figures, data, logs, etc. Make sure to set appropriate paths.
+> ## Set the number of tasks
+>
+> If you run out of memory, try setting ``max_parallel_tasks`` to 1. Then, check the amount of memory you need for that by inspecting the file ``run/resource_usage.txt`` in the output directory. Using the number there you can increase the number of parallel tasks again to a reasonable number for the amount of memory available in your system.
+{: .callout}
 
-.. code-block:: yaml
+## Destination directory
 
+The destination directory is the rootpath where ESMValTool will store its output, i.e. figures, data, logs, etc. With every run, ESMValTool automatically generates a new output folder determined by the recipe name, and date and time using the format: YYYYMMDD_HHMMSS. This folder contains four further subfolders: ``plots``, ``preproc``, ``run``, ``work``.
 
+Let's name our destination directory as ``esmvaltool_output`` in the working directory:
```

Comment:
I would rephrase to "... our destinateion directory ``esmvaltool_output`` in ..."